platform/ui/src/components/StudyListPagination/StudyListPagination.js

```diff
@@ -2,27 +2,28 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Button, ButtonGroup, Typography, Select } from '../';
 import { useTranslation } from 'react-i18next';
+import ExtensibilityService from '../../../../core/src/services/ExtensibilityService/ExtensibilityService';
 
-const StudyListPagination = ({
+function StudyListPaginationFunction({
   onChangePage,
   currentPage,
   perPage,
   onChangePerPage,
-}) => {
-  const { t } = useTranslation("StudyList")
+  extensibility,
+}) {
+  const { t } = useTranslation('StudyList');
 
   const navigateToPage = page => {
     const toPage = page < 1 ? 1 : page;
     onChangePage(toPage);
   };
 
-  const ranges = [
-    { value: '25', label: '25' },
-    { value: '50', label: '50' },
-    { value: '100', label: '100' },
-  ];
-  const [selectedRange, setSelectedRange] = useState(ranges.find(r => r.value === perPage));
-  const onSelectedRange = (selectedRange) => {
+  extensibility = extensibility || StudyListPaginationLevel;
```

Comment:
@swederik As you requested, I've changed the parameter to be a passed in parameter, rather than accessing this.  I think that actually works really well, because the parameter doesn't depend at all on it being from the extensibility service, but is really just a bag of configuration items that can be whatever the parent chooses to pass in.  The only use of the actual service here is to register a default version which comes from the extensibility service.